# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini
# Commit: e0dc13dfef60
# Full Hash: e0dc13dfef60d63093b9823ba18d69e9bb690a1a
# Author: Alexandra Borovova <aborovova@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - Update test expectations for streams/readable-byte-streams/patched-global.any.js. CLOSED TREE
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230568
# ==============================================================================

diff -r c34fddbfd51b -r e0dc13dfef60 testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini
--- a/testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini	Thu Nov 28 16:46:27 2024 +0000
+++ b/testing/web-platform/meta/streams/readable-byte-streams/patched-global.any.js.ini	Thu Nov 28 17:50:38 2024 +0000
@@ -19,20 +19,13 @@
 
 
 [patched-global.any.serviceworker.html]
-  expected:
-    if debug and (os == "win") and (processor == "x86"): [ERROR, CRASH]
-    if debug and (os == "linux") and not fission: [ERROR, CRASH]
-    if not debug: OK
-    ERROR
+  expected: [ERROR, CRASH]
   [Patched then() sees byobRequest after filling all pending pull-into descriptors]
     expected: FAIL
 
 
 [patched-global.any.worker.html]
-  expected:
-    if debug and fission: OK
-    if not debug: OK
-    [OK, CRASH]
+  expected: [OK, CRASH, ERROR]
   [Patched then() sees byobRequest after filling all pending pull-into descriptors]
     expected: FAIL
 
@@ -50,7 +43,9 @@
     [ERROR, CRASH]
 
 [patched-global.https.any.shadowrealm-in-audioworklet.html]
-  expected: ERROR
+  expected:
+    if os == "linux": [ERROR, CRASH]
+    ERROR
 
 [patched-global.any.shadowrealm-in-shadowrealm.html]
   expected:
